doc/source/skips/1-governance.rst

```diff
@@ -145,13 +146,13 @@ are made according to the following rules:
   sentence (but no change of the scikit-image.org landing page or the “about”
   page), require approval by a core developer *and* no disagreement or requested
   changes by a core developer on the issue or pull request page (lazy
-  consensus). Core developers are expected to give “reasonable time” to others
-  to give their opinion on the pull request if they’re not confident others
-  would agree.
+  consensus). Generally, an objection period of one or two days should be given before
+  lazy consensus is assumed.
```

Comment:
```suggestion
  consensus). Core developers are expected to wait one or two days for others
  to give their opinion on the pull request if they’re not confident others
  would agree.
```

Reply:
I like this suggestion; some of consensus operates on an understanding of what would cause conflict, and what would be fine to continue with.

Reply:
See https://github.com/scikit-image/scikit-image/pull/7020/commits/e036dec4204f31f69933c6803f3f3580dfae86c4. Languagetool recommend a comma after "confident". :thinking:

Reply:
> Languagetool recommend a comma after "confident"

Hm, maybe because really it stands for "if they’re not confident *that* others would agree."